docs/terms/access-token.md

```diff
@@ -0,0 +1,11 @@
+---
+id: access-token
+title: Access Token
+hoverText: Used in token-based authentication as [bearer tokens](/docs/terms/bearer-token.md) to access protected resources. [RFC 6749 section-1.4](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4)
```

Comment:
I wonder if we should keep some of the metadata from the other pages, such as the `displayed_sidebar`

Reply:
For these file types, I'm following the guidance of [docusaurus-terminology](https://github.com/grnet/docusaurus-terminology)

Reply:
That makes sense, but the gif in the overview looks like it is supposed to show the [sidebar](displayed_sidebar), which they currently do not. I'm guessing these guidelines are for vanilla Docusaurus? Maybe we can ask for guidelines in whatever Docusaurus framework we're using?

Reply:
Hm I see, so I just tested adding `displayed_sidebar: mainSidebar` which matches the glossary and it now preserves the sidebar. It doesn't add a like the gif but keeping the sidebar is good.

Reply:
I think that's fine. Thanks!